syntax = "proto3";

message RequestRsaPublicKey{

}

message ResponseRsaPublicKey{
	string publickey = 1;
}

message RequestRegister {
  string account = 1;
  string password = 2;
}

message RegisterResult {
  uint32 result = 1;
}

message RequestLogin {
  string account = 1;
  string password = 2;
}

message LoginResult {
  uint32 result = 1;
}